source/utils/index.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file contains general utility functions.
+ */
+
+const crypto = window.crypto; // needed for uuid function
```

Comment:
I think this code explains everything - not sure if we need a comment describing what it does.

Reply:
will remove comment